src/pm/hydra/ui/mpich/utils.c

```diff
@@ -1780,6 +1806,7 @@ static struct HYD_arg_match_table match_table[] = {
     {"hostlist", hostlist_fn, hostlist_help_fn},
     {"ppn", ppn_fn, ppn_help_fn},
     {"profile", profile_fn, profile_help_fn},
+    {"show-rank", show_rank_fn, show_rank_help_fn},
```

Comment:
I think this option is fine to add, but the naming alongside `prepend-rank` could be confusing. When I saw the PR title, I thought it might just be a duplicate of that.

Maybe `filter-output-by-rank` or something like that?

Reply:
Or `show-output-from-rank` with a shorthand option to save some typing 😅.

Reply:
How about `output-from` like `mpirun -output-from 0 -n 3 ...`?

Reply:
works for me

Reply:
Updated.